android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -231,22 +236,23 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
     }
 
     @ReactMethod
-    override fun getRedirectUri(promise: Promise) {
+    fun getRedirectUri(promise: Promise) {
         sdkWrapper.getRedirectUri(promise)
     }
 
     @ReactMethod
-    override fun handleRegistrationCallback(uri: String?) {
+    fun handleRegistrationCallback(uri: String?) {
+        //TODO: Add the nullchecks here and make params nullable
```

Comment:
and missed that one too

Reply:
This function is not promise based yet, so we can't really add the nullchecks here.